From 4135c411af419f4f86dc9ec9301c88012d616c49 Mon Sep 17 00:00:00 2001
From: Romain Vimont <rom@rom1v.com>
Date: Sat, 25 Nov 2023 23:53:30 +0100
Subject: [PATCH] Fix compilation error

Fix the following warning/error:

    ../app/src/cli.c:2158:17: warning: a label can only be part of a
    statement and a declaration is not a statement [-Wpedantic]

With some compilers, this is an error rather than a pedantic warning.

Refs <https://github.com/Genymobile/scrcpy/issues/2256#issuecomment-1467008307>
---
 app/src/cli.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/app/src/cli.c b/app/src/cli.c
index f57b75efc..fd4525f55 100644
--- a/app/src/cli.c
+++ b/app/src/cli.c
@@ -2154,7 +2154,7 @@ parse_args_with_getopt(struct scrcpy_cli_args *args, int argc, char *argv[],
                     return false;
                 }
                 break;
-            case OPT_ORIENTATION:
+            case OPT_ORIENTATION: {
                 enum sc_orientation orientation;
                 if (!parse_orientation(optarg, &orientation)) {
                     return false;
@@ -2162,6 +2162,7 @@ parse_args_with_getopt(struct scrcpy_cli_args *args, int argc, char *argv[],
                 opts->display_orientation = orientation;
                 opts->record_orientation = orientation;
                 break;
+            }
             case OPT_RENDER_DRIVER:
                 opts->render_driver = optarg;
                 break;
